Code/Framework/AzManipulatorTestFramework/Source/RetainedModeActionDispatcher.cpp

```diff
@@ -117,7 +117,7 @@ namespace AzManipulatorTestFramework
 
     RetainedModeActionDispatcher* RetainedModeActionDispatcher::Execute()
     {
-        Log("Executing %u actions", m_actions.size());
+        Log("%s", "Executing %u actions", m_actions.size());
```

Comment:
Isn't the previous version of this the correct one?

Reply:
great catch, yes, my mistake, reverting this